scripts/createDB.py

```diff
@@ -26,6 +26,19 @@
 );
 ''')
 
+# Add info for better calculations
+TABLES['ratings']=(
+'''
+CREATE TABLE IF NOT EXISTS `ratings` (
```

Comment:
I think we talked about supporting rating history right? I think we can have the fields be 
`user_id`
`rating`
`date` 

and add a new entry whenever a user plays a rated run. 

We can scrap `num_rounds` since we can just query for the number of entries of a user in the ratings table. Lmk what you think - one con would definitely be more storage used.

To make computation on ratings easier, maybe we can also maintain a table `current_ratings` that have the most up-to-date rating for a user. Its fields can just be 2 FKs
`user_id`
`rating_id`

So update both tables on a new rating update.

Reply:
Yeah, just having a date attached to the rating is what I was thinking as well. However, I don't know if we can do that with our current ranking system: if someone goes back and plays an old ranked prompt it'll affect everyone. So I think I'm gonna try and implement the prompt of the day stuff first, and just use those for the ratings.